assets/js/components/Dashboard/ScreenDetail.tsx

```diff
@@ -60,11 +61,23 @@ const ScreenCard = (props: ScreenDetailProps) => {
     }
   };
 
-  const getScreenLocation = isPaess
-    ? `/ ${getPaessRoute(paessRouteLetter)}`
-    : screens[0].location
-    ? `/ ${screens[0].location}`
-    : "";
+  const getTriptychPlatform = (directionId: number) =>
+    directionId === 0 ? "Southbound" : "Northbound";
```

Comment:
We'll need to revisit these labels if we ever put the triptych app on other lines (let alone the Orange Line... 😮‍💨) but this works for now.

Reply:
I thought about that too but decided to leave it with OL directions for now. I guess it wouldn't hurt to go ahead and do that now, right?

Reply:
I went ahead and did it. Knowing how the rollout for OL triptychs went, we are likely going to forget about this tiny detail when adding our app to other routes (assuming that ever happens). I'd rather solve it now while it's fresh on my mind.